### PR TITLE
add main page layout with new design #154

### LINKED ISSUE #154: Main page layout + dev deploy
https://www.figma.com/file/Yb10NEuE6IVRPvGVEL8qLZ/%F0%9F%8E%B2-GameJutsu_design?node-id=0%3A1&t=Ov1g0i02RrwV449Y-1

### PR DESCRIPTION
**Pull request by pixelPepe**
* main page layout
Closes #154

### CODE REVIEW

File: components/Header/Header.module.scss [hunk 1]
```diff
@@ -36,3 +36,16 @@
...
+  color: #FFFFFF;
+  background:  linear-gradient(225.67deg, #8DC8E9 3.57%, #8C5AE3 115.97%), #5F5AFA;
+  box-shadow: 0px 4px 12px rgba(0, 0, 0, 0.1);
+  border-radius: 14px;
+  margin-right: 10px;
+
+  @media (max-width: 576px) {
+    font-size: 12px;
+  }
+}
```
Comment (utgarda) [inline components/Header/Header.module.scss:51]: .

File: components/Header/Header.tsx [hunk 2]
```diff
@@ -26,8 +26,63 @@ export const Header: React.FC<HeaderPropsI> = ({version}) => {
         <div className={styles.version}>{`Ver.${version}`}</div>
       </div>
       <div className={styles.right}>
-        {currentPath?.split('?')[0] !== 'connect' && <Navigation active={currentPath} />}
-        {currentPath?.split('?')[0] !== 'connect' && <ConnectButton />}
+        {/* {currentPath?.split('?')[0] !== 'connect' && <Navigation active={currentPath} />} */}
```
Comment (utgarda) [inline components/Header/Header.tsx:29]: .

File: pages/index.tsx [hunk 3]
```diff
@@ -1,56 +1,280 @@
+/* eslint-disable @next/next/no-img-element */
 import type { GetStaticProps, NextPage } from 'next';
 import Head from 'next/head';
 import Image from 'next/image';
 import { useTranslation } from 'react-i18next';
 import styles from '../styles/Home.module.scss';
+// import styles from './/Home.module.scss';
```
Comment (utgarda) [inline pages/index.tsx:7]: .

File: public/images/serveless-communication.png
Comment (utgarda) [inline public/images/serveless-communication.png:1]: Use the provided SVG please.

**Review by anastasiabusygina — COMMENTED** [review]
Верстка:
photo_2023-04-23_13-41-25 (2)
photo_2023-04-23_13-41-25 (3)
photo_2023-04-23_13-41-25 (4)
photo_2023-04-23_13-41-25



Дизайн по макету: 
image

File: public/images/slider1.png
Comment (utgarda) [inline public/images/slider1.png:1]: Do we have SVG files for these?

File: public/images/the-graph-logo.png
Comment (utgarda) [inline public/images/the-graph-logo.png:1]: https://www.figma.com/community/file/1135487878943754077/Logo-GRT-svg-(The-Graph)

**Review by utgarda — APPROVED** [review]
LGTM
take care of cosmetic issues and SVG images, then merge.

File: components/Header/Header.tsx (hunk 2 above)
Comment (pixelPepe) [reply components/Header/Header.tsx]: useful in the future for layout

**Pull request merged**